en/simulation/failsafes.md

```diff
@@ -0,0 +1,22 @@
+# Simulate Failsafes
+
+It's often useful to test safety critical behavior in simulation before taking any risk with a real world scenario. While in HITL simulation the configuration parameters of your real flight control unit is used, in SITL simulation most failsafes are disabled by default to enable quick and easy simulation usage. Also for example the virtual battery is implemented to never run out of energy.
+
+## Link Loss
+By default unavailability of any externel data via MAVLink or directly from a remote remote control (RC) is ignored to make the simulation conveniently usable without additonal dependencies.
+
+- To test RC loss set the parameter `NAV_RCL_ACT` ...
+- To test data link loss set the parameter `NAV_DLL_ACT` ...

```

Comment:
Suggest we break out "Manual control loss" and "Data link loss" as two separate topics since they are two separate failsafes.

Reply:
Sure, makes sense. One mom.

Reply:
Done